edx/edx-enterprise
# create ecommerce order data upon successful manual learner course enrollment

**Description:** Create ecommerce order record for successful enrollment of a learner through the `manage_learners` admin page.

**JIRA:** https://openedx.atlassian.net/browse/ENT-2081

**Dependencies:** https://github.com/edx/ecommerce/pull/2520

**Sandbox:**
- [Manage Learners Admin Page](https://mammar.sandbox.edx.org/admin/enterprise/enterprisecustomer/cc8a0260-36d0-4520-8e1a-154d0431aa9c/change/)
- [Ecommerce Order Page](https://ecommerce-mammar.sandbox.edx.org/dashboard/orders/)

**Testing:**
1. Go to manage learners page
2. Enroll single or multiple learners. Enrollment should be successful.
3. Go to ecommerce orders page.
4. You should see an order with correct data for the learner enrolled.

**Single Enrollment Order Creation Failure Message**
<img width="1439" alt="Screenshot 2019-08-21 at 4 45 08 PM" src="https://user-images.githubusercontent.com/6767924/63429640-10c1f180-c434-11e9-8abf-479710278de0.png">


**Bulk Enrollment Order Creation Failure Message**
<img width="1440" alt="Screenshot 2019-08-21 at 4 48 17 PM" src="https://user-images.githubusercontent.com/6767924/63429655-1c151d00-c434-11e9-9c9f-80bc2773d228.png">
